src/test/scala/sigmastate/utxo/AVLTreeScriptsSpecification.scala

```diff
@@ -33,9 +35,9 @@ class AVLTreeScriptsSpecification extends SigmaTestingCommons {
     val treeData = new AvlTreeData(digest, 32, None)
 
     val env = Map("key" -> key, "proof" -> proof)
-    val prop = compile(env, """isMember(SELF.R3[AvlTree].value, key, proof)""").asBoolValue
+    val prop = compile(env, """isMember(SELF.R4[AvlTree].value, key, proof)""").asBoolValue
```

Comment:
@kushti we should probably be able to write script like this?
`SELF.reg[AvlTree](reg1).value`